data/avgi/avgi ships.txt

```diff
@@ -0,0 +1,2142 @@
+	attributes
+		category "Utility"
+		licenses
+			"Avgi Civilian"
+		"shields" 2400
+		"hull" 15000
+		"required crew" 12
+		"bunks" 60
+		"cargo space" 600
+		"mass" 600
+		"drag" 4.2
+		"heat dissipation" 1.8
+		"fuel capacity" 2400
+		"outfit space" 630
+		"weapon capacity" 153
+		"engine capacity" 168
+		"spinal mount" 1
+		"anchor point" 3
+			"blast radius" 540
+			"shield damage" 3600
+			"hull damage" 2400
+			"hit force" 4800
+	outfits
+		"Phased Array Laser (Green)" 3
+		"Speckle Turret" 2
+		"Speck Round" 14400
+		"Sidereal Launcher" 2
+		"Sidereal Missile" 84
+		"Electromotive Rifle" 24
+
+		"Luminous Fusion Core"
+		"Ultracapacitor Bank" 3
+		"Diffuse Deflector"
+		"Optical Lasing Generator"
+		"Electrostatic Ramscoop"
+		"Multispectral Scanner"
+		"Spectrometer Array"
+
+		"Z-360 Fusion Torch"
+		"XY-420 Radial Thrusters"
+		"Hyperdrive"
+
+	gun 0 -213
+		over
+	gun -42 -139 "Sidereal Launcher"
+		angle -90
+	gun 42 -139 "Sidereal Launcher"
+		angle 90
+	turret -33 -138 "Speckle Turret"
+	turret 33 -138 "Speckle Turret"
+	turret 0 -30 "Phased Array Laser (Green)"
+	turret 0 42 "Phased Array Laser (Green)"
+	turret 0 186 "Phased Array Laser (Green)"
+
+	bay "Drone" -65.5 3
+	bay "Drone" 65.5 3
+	bay "Drone" 65.5 29
+	bay "Drone" -65.5 29
+
+	engine 0 239
+	"reverse engine" -6 240
+		angle -30
+	"reverse engine" 6 240
+		angle 30
+	"steering engine" -42 -138
+		angle 90
+		right
+	"steering engine" 42 -138
+		angle -90
+		left
+	"steering engine" -6 240
+		angle 90
+		left
+	"steering engine" 6 240
+		angle -90
+		right
+
+	leak "leak" 30 90
+	leak "flame" 30 90
+	leak "big leak" 30 90
+	explode "tiny explosion" 18
+	explode "small explosion" 30
+	explode "medium explosion" 60
+	explode "large explosion" 30
+	explode "huge explosion" 9
+	"final explode" "final explosion large"
+
+	description "The Undsyni-class Auxiliary Freighter is not just a bulk freighter, but one built to be large and strong enough to hold its own against attacks from smaller, more militarized vessels. While not quite as payload efficient as more spindly ships such as the Avlaki and Harmonikos, the Undsyni's resilience makes it much more survivable against attacks, with its fusion torch giving it the acceleration to both evade enemies and minimize travel times."
+	description "Even before the Incursion, the Undsyni's strong hull and internal outfit capacity made it a popular option as a capital warship for groups who could not gain access to specialized warships."
+	description "A reinforced spine running the length of the ship allows it to fit a huge Dipole Ramscoop, capable of bearing the extreme stresses the ramscoop's magnetic field puts on its superconducting magnets. In this configuration, a tanker Undsyni can supply the fuel needs of an entire fleetgroup across dozens of systems."
+
+
+ship "Modified Tachytia"
+	sprite "ship/avgi tachytia/tachytia"
+		"frame rate" 30
+	thumbnail "thumbnail/avgi tachytia"
+	swizzle 0
+	attributes
+		category "Utility"
```

Comment:
```suggestion
		category "Utility"
		"cost" 72000000
```
Roughly the same order of magnitude as other very large ships. Admittedly largely irrelevant, as presumably this ship will never be sold in shipyards.